modules/civicrm_group_roles/civicrm_group_roles.module

```diff
@@ -330,45 +330,45 @@ function civicrm_role_granted_by_other_group($contact_id, $rid, $group_id) {
 function civicrm_group_roles_menu() {
   $items = array();
   $items['admin/config/civicrm/civicrm_group_roles'] = array(
-    'title' => t('CiviGroup Roles Sync'),
-    'description' => t('Add/remove association rules and configure settings. Also perform manual synchronization.'),
+    'title' => 'CiviGroup Roles Sync',
+    'description' => 'Add/remove association rules and configure settings. Also perform manual synchronization.',
     'page callback' => 'civicrm_group_roles_show_rules',
     'access callback' => 'user_access',
     'access arguments' => array('access civicrm group role setting'),
     'type' => MENU_NORMAL_ITEM,
   );
   $items['admin/config/civicrm/civicrm_group_roles/show_rules'] = array(
-    'title' => t('Association rules'),
+    'title' => 'Association rules',
     'access callback' => 'user_access',
     'access arguments' => array('access civicrm group role setting'),
     'weight' => -5,
     'type' => MENU_DEFAULT_LOCAL_TASK,
   );
   $items['admin/config/civicrm/civicrm_group_roles/add_rule'] = array(
-    'title' => t('Add association rule'),
```

Comment:
See https://drupal.stackexchange.com/questions/109715/should-i-use-t-in-a-hook-menus-description for why this is doesn't need t